docs/github/v2/03-test-runner.md

```diff
@@ -159,7 +159,7 @@ _**default:** `<your project root>`_
 
 #### customParameters
 
-Custom parameters to configure the test runner.
+Unity Test Framework [Custom parameters](https://docs.unity3d.com/Packages/com.unity.test-framework@2.0/manual/reference-command-line.html) to configure the test runner.
```

Comment:
This isn't quite correct though.

They're custom parameters that are passed to the editor directly. They don't have to be from any framework per se. Test framework custom parameters are merely an example of what you can pass (and won't even work if you don't have that package installed I think).